### DISCUSSION TITLE
Unzip (LJSpeech dataset) to the data folder and upsample the data to 24 kHz

### DISCUSSION THREAD

**Discussion by enoky** [yl4579/StyleTTS2#107] [Q&A]
Hi, I am setting up StyleTTS2 to run locally on Windows 11 in PyCharm. I have all prerequisites installed. What is the best method to batch process the WAV files in the LJSpeech dataset to 24kHz? I have been running Coqui TTS in PyCharm and would like to test StyleTTS2's capabilities since the demos sound much more natural.

EDIT: I found a way to batch upsample the dataset to 24kHz using SoX and a PowerShell script found at the link below. Simply edit the PowerShell script so it produces the correct sample rate.

https://www.elektronauts.com/t/quick-tutorial-for-windows-10-batch-sample-conversion-using-sox/145776

**Comment by pitxels** [comment]
For MAC I found [sox-o-matic](https://www.birds.cornell.edu/ccb/sox-o-matic/)
